tests/unit/test_apigateway.py

```diff
@@ -208,24 +208,21 @@ def test_render_template_values():
     assert decoded == "x=a b"
 
     escape_tests = (
-        ("it's", '"it\'s"'),
-        ("0010", "10"),
+        ("it's", "it's"),
+        ("0010", "0010"),
         ("true", "true"),
-        ("True", '"True"'),
+        ("True", "True"),
         ("1.021", "1.021"),
-        ("'''", "\"'''\""),
-        ('""', '""'),
-        ('"""', '"\\"\\"\\""'),
-        ('{"foo": 123}', '{"foo": 123}'),
-        ('{"foo"": 123}', '"{\\"foo\\"\\": 123}"'),
-        (1, "1"),
-        (True, "true"),
+        ('""', '\\"\\"'),
+        ('"""', '\\"\\"\\"'),
+        ('{"foo": 123}', '{\\"foo\\": 123}'),
+        ('{"foo"": 123}', '{\\"foo\\"\\": 123}'),
+        (1, 1),
+        (None, None),
```

Comment:
Shouldn't this be
```suggestion
        (None, "null"),
```
?

Reply:
The `None` is not a string, so not sure what you are referring to. 

This is what happens on AWS:

Input:
```json
{
    "myAttr": null
}
```

With the transformation template:

```
{
  "result": $util.escapeJavaScript($input.json('$.myAttr'))
}
```

The result:

```
Endpoint request body after transformations: {
  "result": null
}
```

Reply:
Great, thanks for checking the behavior on real AWS @calvernaz ! 🙌  Again, sorry for not making myself clear - what I wanted to suggest is that we should make `escapeJavaScript` return a `str` in any case (i.e., should not return values like `None`, or the number `0`, etc). 

I've pushed a tiny commit to ensure `escapeJavaScript` returns a string (and also added a corresponding type hint) - hope that's ok for you. Excited to get this merged soon! 🎉

Reply:
I think i did, but the string "null" is not the null javascript value or json fwiw. So IMO that should not be handled by this function but from the caller.